cosmo_tester/test_suites/snapshots/multi_tenant_snapshot_test.py

```diff
@@ -13,141 +13,182 @@
 #    * See the License for the specific language governing permissions and
 #    * limitations under the License.
 
-import json
-import os
-import uuid
-
 import pytest
 
-from cosmo_tester.framework.cluster import (
-    CloudifyCluster,
-    MANAGERS,
-)
-from cosmo_tester.framework.util import assert_snapshot_created
-
-from cloudify_cli.constants import DEFAULT_TENANT_NAME
-
 from . import (
-    deployment_id,
-    deploy_helloworld,
-    wait_for_execution,
-    update_client_tenant
+    assert_hello_worlds,
```

Comment:
The problem is that the single and multi tenants both make use of these but with differences that I think would overcomplicate any general high level method shared between them- it would of course be possible to generalise it, but I think the complexity of the shared method would outweigh the gains.